py/desisim/scripts/quickquasars.py

```diff
@@ -193,11 +193,11 @@ def simulate_one_healpix(ifilename,args,model,obsconditions,decam_and_wise_filte
     pixel, nside, hpxnest = get_healpix_info(ifilename)
 
     # using global seed (could be None) get seed for this particular pixel
-    #global_seed = args.seed
-    #seed = get_pixel_seed(pixel, nside, global_seed)
+    global_seed = args.seed
+    seed = get_pixel_seed(pixel, nside, global_seed)
     # use this seed to generate future random numbers
     seed=args.seed
```

Comment:
@alxogm - you have to delete this last line (199) that you added by mistake, since we dont want to use args.seed as the seed for all pixels, we want to use what get_pixel_seed returns.